sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentimentScorePerLabel.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The SentimentScorePerLabel model
+ */
+@Immutable
+public final class SentimentScorePerLabel {
+    private final double negativeScore;
+    private final double neutralScore;
+    private final double positiveScore;
+
+    /**
+     * Creates a {@code SentimentScorePerLabel} model that describes the sentiment score of the sentiment label.
+     *
+     * @param negativeScore negative score value, range in between 0 and 1.0
+     * @param neutralScore neutral score value, range in between 0 and 1.0
+     * @param positiveScore positive score value, range in between 0 and 1.0
+     */
+    public SentimentScorePerLabel(double negativeScore, double neutralScore, double positiveScore) {
+        this.negativeScore = negativeScore;
```

Comment:
Consider eliminating `Score` from the names. Our users will do i.e.
`xx.getSentimentScore().getPositive()` so it already implies positive is a score.

Reply:
It will also align with other languages